Sources/Common/Pagination/Paginator.swift

```diff
@@ -45,15 +49,21 @@ public struct Paginator<Item: Hashable & Identifiable, Flow: IdentifiableFlow>:
             }
             
         case let action as Actions.LoadPage where action.id == Flow.id && action.pageNumber == initialPage:
+            isLoading = true
             page = .number(initialPage)
             
         case let action as Actions.LoadPage where action.id == Flow.id:
             guard case .number = self.page else {
                 return
             }
-            
+
             page = .number(action.pageNumber)
-            
+        case let action as Actions.Error where action.id == Flow.id:
+            isLoading = false
+
+            if page.pageNumber > initialPage {
```

Comment:
```suggestion
            if page.pageNumber > initialPage, isLoading {
```

А сам isLoading = false ставить после if'a